x-pack/plugins/osquery/cypress/integration/superuser/alerts.spec.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ArchiverMethod, runKbnArchiverScript } from '../../tasks/archiver';
+import { login } from '../../tasks/login';
+import {
+  checkResults,
+  findAndClickButton,
+  findFormFieldByRowsLabelAndType,
+  inputQuery,
+  submitQuery,
+} from '../../tasks/live_query';
+import { preparePack } from '../../tasks/packs';
+import { closeModalIfVisible } from '../../tasks/integrations';
+import { navigateTo } from '../../tasks/navigation';
+
+describe('Alert Event Details', () => {
+  before(() => {
+    runKbnArchiverScript(ArchiverMethod.LOAD, 'pack');
+    runKbnArchiverScript(ArchiverMethod.LOAD, 'rule');
+  });
+  beforeEach(() => {
+    login();
+  });
+
+  after(() => {
+    runKbnArchiverScript(ArchiverMethod.UNLOAD, 'pack');
+    runKbnArchiverScript(ArchiverMethod.UNLOAD, 'rule');
+  });
+
+  it('should be able to run live query', () => {
```

Comment:
Can you add tests for potential failure scenarios as well. To make sure we fail gracefully?

Reply:
I've been trying to think of a scenario that meets these expectations, and couldn't really find any. Would you like to see a test that has a osquery failing? Or OsquerActionItem not available? 

Thanks!